### PR TITLE
Documentation for Haxe Programming Language support.

### PR DESCRIPTION
**Pull request by andyli**
See travis-ci/travis-build#406 and travis-ci/travis-core#447.

### CODE REVIEW

File: user/languages/haxe.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+for example:
+
+    language: haxe
+    neko: "2.0.0"
+
+However, unlike `haxe:`, you can provide only one value (not an array) to `neko:`.
+
+## Test Configuration
+
+If your project make use of the standard hxml files for building, you can specify
```
Comment (Simn) [inline user/languages/haxe.md]: Small grammar issue: either "projects make" or "project makes"

Comment (andyli) [inline user/languages/haxe.md]: Just fixed! Thanks for proofreading!

**Comment by BanzaiMan** [comment]
The matrix expansion part needs to wait a bit.

**Comment by BanzaiMan** [comment]
Matrix expansion support has been deployed now.

**Pull request merged**